services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -4702,7 +4672,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/put/responses/200/content/application~1json/schema'
+                $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/get/responses/200/content/application~1json/schema'
```

Comment:
This changed from `put` to `get`, is this as expected? Same happened above.

Reply:
will double check looks weird

Reply:
it's actually correct. it was wrong before. the operation is a get.